snapcraft/yaml.py

```diff
@@ -308,10 +309,19 @@ def _validate_snapcraft_yaml(snapcraft_yaml):
 
 def _snapcraft_yaml_load(yaml_file='snapcraft.yaml'):
     try:
-        with open(yaml_file) as fp:
-            return yaml.load(fp)
+        with open(yaml_file, 'rb') as fp:
+            bs = fp.read(2)
     except FileNotFoundError:
```

Comment:
I'll look into this one in a different PR, it has more repercussions and work than this particular fix :wink: